File: JAVA/Daily Flash Codes/Week2/Day5_Jan24/code/prog4.java


import java.io.*;

class Program {
	public static void main(String[] args) throws IOException {
		BufferedReader in = new BufferedReader(new InputStreamReader(System.in));		//obj of Bufferedreader to take input
		int row;
		System.out.println("Enter row: ");
		row = Integer.parseInt(in.readLine());
		
		for(int olc=0; olc<row; olc++) {
			for(int ilc=0; ilc<=olc; ilc++) {
				if(olc%2==0)
					System.out.print(" 0 ");
				else
					System.out.print(" 1 ");
			}
			System.out.println();
		}

	

	}


}

File: C++/Daily Flash/Week2/Day4_Jan23/code/prog4.cpp
/*Program 4: Write a Program to Print following Pattern
*
* * 
* * *
* * * * 

*/



#include <iostream>

int main () {
	int row;
	std::cout << "Enter row: ";
	std::cin >> row;

	for(int olc = 0; olc<row; olc++) {

		for(int ilc = 0; ilc<=olc; ilc++) {
		        std::cout << " * ";	
		}
		std::cout << std::endl;

	}
	return 0;
}
